webviewer/src/pages/Account/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,31 @@
+import { Icon } from 'components/_shared/Icon';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { ROUTES } from '_constants';
+import { StyledSidebar, BreadCrumbsItem, BreadCrumbs, TabItem } from './Sidebar.styles';
+
+export const Sidebar = props => {
+  const { activeTab, setActiveTab, tabs } = props;
+  const history = useHistory();
+  const handleClickHome = () => history.push(ROUTES.ROOT);
+
+  return (
+    <StyledSidebar>
+      <BreadCrumbs>
+        <BreadCrumbsItem onClick={handleClickHome}>Home</BreadCrumbsItem> /{' '}
+        <BreadCrumbsItem>Personal account</BreadCrumbsItem>
```

Comment:
please remove `cursor:pointer` from the last/active breadcrumb

Reply:
fixed